test/e2e/common/podtemplates.go

```diff
@@ -31,8 +31,13 @@ import (
 
 var _ = ginkgo.Describe("[sig-architecture] PodTemplates", func() {
 	f := framework.NewDefaultFramework("podtemplate")
-
-	ginkgo.It("should run the lifecycle of PodTemplates", func() {
+	/*
+	   Release : v1.18
+	   Testname: ConfigMap lifecycle
```

Comment:
Description is written for ConfigMap not PodTemplate

Reply:
Updated the metadata in 97fdf67